### PR TITLE
XRDDEV-1661 Metapackage install restarts

### PR DESCRIPTION
**Pull request by carohauta**
JIRA: https://jira.niis.org/browse/XRDDEV-1661

### CODE REVIEW

File: src/packages/src/xroad/ubuntu/generic/xroad-securityserver-fi.postinst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ "$1" = configure ]; then
+  invoke-rc.d --quiet xroad-proxy try-restart || true
+  invoke-rc.d --quiet xroad-signer try-restart || true
+  invoke-rc.d --quiet xroad-addon-messagelog try-restart || true
```
Comment (jhyoty) [inline src/packages/src/xroad/ubuntu/generic/xroad-securityserver-fi.postinst:6]: messagelog is not a separate service (xroad-proxy plugin), no need to restart it.

File: src/packages/src/xroad/redhat/SPECS/xroad-securityserver-fi.spec [hunk 2]
```diff
@@ -37,3 +37,7 @@ cp -p %{srcdir}/default-configuration/override-securityserver-fi.ini %{buildroot
 
 %post
 
+%postun
+if [ $1 -gt 0 ] ; then
```
Comment (jhyoty) [inline src/packages/src/xroad/redhat/SPECS/xroad-securityserver-fi.spec:41]: redundant test, the macro already checks for upgrade

File: src/packages/src/xroad/ubuntu/generic/xroad-securityserver-is.postinst [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [ "$1" = configure ]; then
+  invoke-rc.d --quiet xroad-addon-messagelog try-restart || true
```
Comment (jhyoty) [inline src/packages/src/xroad/ubuntu/generic/xroad-securityserver-is.postinst:4]: If there are messagelog customizations, one needs to restart xroad-proxy (there is no xroad-addon-messagelog service)

File: src/packages/src/xroad/redhat/SPECS/xroad-securityserver-is.spec [hunk 4]
```diff
@@ -36,3 +36,8 @@ cp -p %{srcdir}/default-configuration/override-securityserver-is.ini %{buildroot
 %upgrade_check
 
 %post
+
+%postun
+if [ $1 -gt 0 ] ; then
+    %systemd_postun_with_restart xroad-addon-messagelog.service
```
Comment (jhyoty) [inline src/packages/src/xroad/redhat/SPECS/xroad-securityserver-is.spec:42]: should be `xroad-proxy`; there is no messagelog service

File: src/packages/src/xroad/redhat/SPECS/xroad-securityserver-fi.spec [hunk 5]
```diff
@@ -37,3 +37,7 @@ cp -p %{srcdir}/default-configuration/override-securityserver-fi.ini %{buildroot
...
+    %systemd_postun_with_restart xroad-signer.service xroad-proxy.service xroad-addon-messagelog.service
```
Comment (jhyoty) [inline src/packages/src/xroad/redhat/SPECS/xroad-securityserver-fi.spec:42]: there is no messagelog service

File: src/packages/src/xroad/ubuntu/generic/xroad-securityserver-fi.postinst (hunk 1 above)
Comment (carohauta) [reply src/packages/src/xroad/ubuntu/generic/xroad-securityserver-fi.postinst]: Fixed this to all

File: src/packages/src/xroad/redhat/SPECS/xroad-securityserver-fi.spec (hunk 2 above)
Comment (carohauta) [reply src/packages/src/xroad/redhat/SPECS/xroad-securityserver-fi.spec]: So it seems, yeah. Removed the checks from all spec-files.

File: src/packages/src/xroad/ubuntu/generic/xroad-securityserver-is.postinst (hunk 3 above)
Comment (carohauta) [reply src/packages/src/xroad/ubuntu/generic/xroad-securityserver-is.postinst]: Fix'd

File: src/packages/src/xroad/redhat/SPECS/xroad-securityserver-is.spec (hunk 4 above)
Comment (carohauta) [reply src/packages/src/xroad/redhat/SPECS/xroad-securityserver-is.spec]: Fix'd

File: src/packages/src/xroad/redhat/SPECS/xroad-securityserver-fi.spec (hunk 5 above)
Comment (carohauta) [reply src/packages/src/xroad/redhat/SPECS/xroad-securityserver-fi.spec]: Fix'd

**Review by jhyoty — APPROVED** [review]

**Pull request merged**